build.sbt

```diff
@@ -654,108 +669,111 @@ lazy val `scio-google-cloud-platform`: Project = project
-      "com.google.cloud.bigtable" % "bigtable-client-core" % bigtableClientVersion,
+      // compile
       "com.chuusai" %% "shapeless" % shapelessVersion,
+      "com.google.api" % "gax" % gaxVersion,
+      "com.google.api" % "gax-grpc" % gaxVersion,
       "com.google.api-client" % "google-api-client" % googleClientsVersion,
-      "com.google.api.grpc" % "proto-google-cloud-bigquerystorage-v1beta2" % bigQueryStorageBetaVersion,
       "com.google.api.grpc" % "proto-google-cloud-bigquerystorage-v1" % bigQueryStorageVersion,
+      "com.google.api.grpc" % "proto-google-cloud-bigquerystorage-v1beta2" % bigQueryStorageBetaVersion,
       "com.google.api.grpc" % "proto-google-cloud-bigtable-admin-v2" % googleCloudBigTableVersion,
       "com.google.api.grpc" % "proto-google-cloud-bigtable-v2" % googleCloudBigTableVersion,
-      "com.google.api" % "gax-grpc" % gaxVersion,
-      "com.google.api" % "gax" % gaxVersion,
       "com.google.apis" % "google-api-services-bigquery" % googleApiServicesBigQueryVersion,
       "com.google.auth" % "google-auth-library-credentials" % googleAuthVersion,
       "com.google.auth" % "google-auth-library-oauth2-http" % googleAuthVersion,
       "com.google.cloud" % "google-cloud-bigquerystorage" % bigQueryStorageVersion,
-      "com.google.cloud" % "google-cloud-core" % googleCloudVersion,
-      "com.google.cloud" % "google-cloud-storage" % googleCloudVersion % "test,it",
+      "com.google.cloud" % "google-cloud-core" % googleCloudCoreVersion,
+      "com.google.cloud" % "google-cloud-spanner" % googleCloudSpannerVersion,
+      "com.google.cloud.bigtable" % "bigtable-client-core" % bigtableClientVersion,
       "com.google.guava" % "guava" % guavaVersion,
-      "com.google.http-client" % "google-http-client-jackson2" % googleHttpClientsVersion,
       "com.google.http-client" % "google-http-client" % googleHttpClientsVersion,
+      "com.google.http-client" % "google-http-client-jackson2" % googleHttpClientsVersion,
       "com.google.protobuf" % "protobuf-java" % protobufVersion,
-      "com.spotify" %% "magnolify-cats" % magnolifyVersion % "test",
-      "org.typelevel" %% "cats-core" % catsVersion % "test",
-      "com.spotify" %% "magnolify-scalacheck" % magnolifyVersion % "test",
       "com.twitter" %% "chill" % chillVersion,
       "commons-io" % "commons-io" % commonsIoVersion,
       "joda-time" % "joda-time" % jodaTimeVersion,
-      "junit" % "junit" % junitVersion % "test",
       "org.apache.avro" % "avro" % avroVersion,
       "org.apache.beam" % "beam-sdks-java-core" % beamVersion,
       "org.apache.beam" % "beam-sdks-java-extensions-google-cloud-platform-core" % beamVersion,
-      "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion,
+      "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion excludeAll (
+        // remove transitive test dependencies at compile time
+        "junit" % "junit",
+        "org.hamcrest" % "hamcrest-core"
```

Comment:
`beam-sdks-java-io-google-cloud-platform` packages some junit test suite in the main artifact.
Just exclude `junit` and `hamcrest-core` (and also, we should prefer `hamcrest` instead of sub modules artifacts)